include/networkit/graph/GraphBuilder.hpp

```diff
@@ -160,9 +176,20 @@ class GraphBuilder {
     void increaseInWeight(node u, node v, edgeweight ew);
 
     /**
-     * Generates a Graph instance. The graph builder will be reseted at the end.
+     * Generates a Graph instance. The graph builder will be resetted at the end.
      */
-    Graph completeGraph(bool parallel);
+    Graph completeGraph();
+
+    /**
+     * DEPRECATED: use completeGraph() instead which uses the parallel mode by default (if
+     * possible).
+     */
+    Graph TLX_DEPRECATED(completeGraph(bool parallel)) {
+        WARN("GraphBuilder::completeGraph(bool parallel) is deprecated, use "
+             "GraphBuilder::completeGraph() instead");
+        if (parallel || !parallel) // otherwise throws "-WERROR-unusedParameter"
```

Comment:
Better use `maybe_unused` for the `parallel` variable: https://en.cppreference.com/w/cpp/language/attributes/maybe_unused